---
title: "The AMP-AD Wall Of Targets"
subtitle: "Decide what to put here"
author: "[Lara Mangravite](lara.mangravite@sagebase.org), [Ben Logsdon](ben.logsdon@sagebase.org),<br/>[Kenneth Daily](mailto:kenneth.daily@sagebase.org)"
date: "2017-07-11"
logo: "img/Sage_neon_logo1.png"
output:
  revealjs::revealjs_presentation:
    theme: night
    template: template.html
    transition: fade
    center: true
    
---

## Goal

Create a visual dashboard and integrative analysis platform that provides consistent and robust supporting evidence for use in:

* Prioritization of targets nominated by individual AMP-AD projects.
* Nomination of targets supported across AMP-AD projects.
* Compilation of AMP-AD evidence for use by research community to evaluate other targets of interest.

## Approach

1. Develop prototype.
1. Build alpha platform based on user feedback, focusing on gene-based
queries for nominated AMP-AD targets.
1. Expand to include genes across the transcriptome (partnership with
Josh Shulman @ Baylor).
1. Expand to provide entry points based on gene, protein, network,
or variant queries.

> **Anticipated users:** AMP-AD consortium and affiliates, MODEL-AD consortium, external AD researchers

## Enhancing AMP-AD Project R01

*Increasing resources for target prioritization<br/>within the AMP-AD community*

![amp steering](img/ampadsteering.png)

## User experience interviews

> Understand how our users will interact and utilize the Wall of Targets application - usages, pain points, gaps, and preferences.

> Performed **think aloud exercise** - use the application and state what you are doing and what you are thinking as you use the prototype. 

> 10 participants, including 4 industry/pharma and 3 mouse model system researchers

## Version 0, main page

![original main page](img/old_mainpage.png)

## Version 0, detail page

![original detail page](img/old_details_SYT1.png)

## Interviews - what we learned

1. Biological context
1. Nominations and source of evidence
1. Consistency and reproducibility across cohorts
1. Druggability vs. tractability
1. Ranking and comparison

## Overview design

![new main page](img/overview.png)

## Main page, alpha version

![new main page](img/new_mainpage.png)

> Includes overview of evidence for indication of strength of target.

## Details page, alpha version

![new detail page](img/new_detailspage_syt1.png)

> Condensed layout including biological context.

## Details page, alpha version

![new detail page](img/new_detailspage_syt1_annotated.png)

> Indication of data sources.

## Gene Ontology terms, alpha version

![new go annotations](img/new_goannot_syt1.png)

## Biological pathways, alpha version

![new biological pathways](img/new_biopathways_syt1.png)

## Mouse models, alpha version

![new mouse models](img/new_mousemodels.png)

## Baseline RNA Expression, alpha version

![new expression atlas](img/new_expressionatlas.png)

> Expression from more sources<br/>and outside of brain.

## Search

![new details page](img/search.jpg)

> Enable for nominated targets<br/>(future: whole transcriptome)

## Nomination evidence, alpha version

![new nomination evidence](img/new_nominationevidence_SYT1.png)

> Show evidence from nominating center(s)

## Looking forward

- Druggability vs. tractability
- Module-centric search and interaction
- eQTL and genomic variants
- Evidence-based ranking

## Evidence-based ranking

![new main page](img/new_mainpage.png)

## 

![ben slide 1](img/ben_Slide1.jpg)

## 

![ben slide 2](img/ben_Slide2.jpg)

## 

![ben slide 3](img/ben_Slide3.jpg)

## Thank you

